Applications/DataExplorer/VtkVis/VtkAlgorithmProperties.cpp

```diff
@@ -22,7 +22,7 @@
 #include <vtkTexture.h>
 
 #include "VtkColorLookupTable.h"
-#include "FileIO/XmlIO/Qt/XmlLutReader.h"
+#include "XmlIO/Qt/XmlLutReader.h"
```

Comment:
Is that a full path from the root of the source directory? If not please correct.